src/sagemaker/config/config_schema.py

```diff
@@ -128,6 +130,10 @@ def _simple_path(*args: str):
     TRAINING_JOB_VPC_CONFIG_PATH, SECURITY_GROUP_IDS
 )
 TRAINING_JOB_SUBNETS_PATH = _simple_path(TRAINING_JOB_VPC_CONFIG_PATH, SUBNETS)
+TRAINING_JOB_PROFILE_CONFIG_PATH = _simple_path(SAGEMAKER, TRAINING_JOB, PROFILER_CONFIG)
+TRAINING_JOB_PROFILE_CONFIG_DISABLE_PROFILER_PATH = _simple_path(
+    SAGEMAKER, TRAINING_JOB, PROFILER_CONFIG, DISABLE_PROFILER
```

Comment:
Can we reuse the TRAINING_JOB_PROFILE_CONFIG_PATH here? 
So it can be TRAINING_JOB_PROFILE_CONFIG_PATH, DISABLE_PROFILER.

Reply:
neat will make the update!